packages/hyper-ui/src/components/purchase-block/index.vue

```diff
@@ -146,8 +146,8 @@ export default {
             default: 0
         },
         releaseDate: {
-            type: String,
-            default: new Date()
+            type: [String, Date],
```

Comment:
This is bad. It should only accept one or the other. It should be the responsibility of the user to pass the correct type.

Reply:
Ok. So what we should use for Date field?

Reply:
String. UI components should be flexible, they shouldn't enforce the way a date is displayed.